From ltsi-dev-bounces@lists.linuxfoundation.org Fri Jul  6 05:10:18 2012
From: Simon Horman <horms@verge.net.au>
Date: Fri,  6 Jul 2012 21:09:57 +0900
Subject: ARM: mach-shmobile: armadillo800eva: defconfig Allow use of armhf userspace
To: ltsi-dev@lists.linuxfoundation.org
Cc: Nobuhiro Iwamatsu <nobuhiro.iwamatsu.yj@renesas.com>
Message-ID: <1341576597-13986-1-git-send-email-horms@verge.net.au>


This allows use a Debian armhf usespace
as well as the existing ability to use a Debian armel userspace.

N.B: Not present upstream yet

Cc: Nobuhiro Iwamatsu <nobuhiro.iwamatsu.yj@renesas.com>
Signed-off-by: Simon Horman <horms@verge.net.au>

---

Hi Greg,

please consider this for inclusion in LTSI.
---
 arch/arm/configs/armadillo800eva_defconfig |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/arch/arm/configs/armadillo800eva_defconfig
+++ b/arch/arm/configs/armadillo800eva_defconfig
@@ -21,7 +21,7 @@ CONFIG_ARCH_SHMOBILE=y
 CONFIG_ARCH_R8A7740=y
 CONFIG_MACH_ARMADILLO800EVA=y
 # CONFIG_SH_TIMER_TMU is not set
-# CONFIG_ARM_THUMB is not set
+CONFIG_ARM_THUMB=y
 CONFIG_CPU_BPREDICT_DISABLE=y
 # CONFIG_CACHE_L2X0 is not set
 CONFIG_ARM_ERRATA_430973=y
@@ -39,6 +39,7 @@ CONFIG_ZBOOT_ROM_BSS=0x0
 CONFIG_CMDLINE="console=tty0 console=ttySC1,115200 earlyprintk=sh-sci.1,115200 ignore_loglevel root=/dev/nfs ip=dhcp nfsroot=,rsize=4096,wsize=4096"
 CONFIG_CMDLINE_FORCE=y
 CONFIG_KEXEC=y
+CONFIG_VFP=y
 # CONFIG_CORE_DUMP_DEFAULT_ELF_HEADERS is not set
 # CONFIG_SUSPEND is not set
 CONFIG_NET=y
